app/assets/stylesheets/marketing/modules/_variables.scss

```diff
@@ -15,7 +15,7 @@ $max-width: 				1001px;
  * Breakpoints
  */
 $short: 					new-breakpoint(max-width 22.5em 4);     // 360px
-$tall: 						new-breakpoint(max-width 47.8em 12);      // 768px
+$tall: 						new-breakpoint(max-width 52.94em 12);      // 847px
```

Comment:
I would be surprised if thats the case, this calculation is the _exact_ width required to keep the logo from "bumping up"